harvard-lts/drs-import-management-service
# End to end integration tests for simple CMs

**End to End Integration tests for the 5 simple content models*
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/ETD-259

# What does this Pull Request do?
Allows for simple end-to-end testing of an ETD in the DAIS pipeline.

# How should this be tested?
Verify old tests pass by following the instructions in the [readme](https://github.com/harvard-lts/drs-import-management-service#testing)

See https://github.com/harvard-lts/etd-integration-tests/pull/15 for associated integration testing

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N
- integration tests? Y
